skeletons/widget-test.js

```diff
@@ -6,7 +6,7 @@ describe("testing a %WIDGET_NAME% widget", function() {
 
   it("should test the %WIDGET_NAME% creator", function() {
     var res = this.createTestWidgetString("%WIDGET_NAME%", {}, "<label>Test</label>");
-    var widget = qx.bom.Html.clean([res[1]])[0];
+    var widget = (function(){var div=document.createElement('div');div.innerHTML=res[1];return div.childNodes[0];})();
```

Comment:
As it is not really clear what this function does, it should be declared somewhere outside the tests with a self explaining name and a little description. That function can be used everywhere, where `qx.bom.Html.clean`got replaced.